README.md

```diff
@@ -3,73 +3,6 @@
 The Azul project contains the components that together serve as the backend to
 Boardwalk, a web application for browsing genomic data sets.
 
-# Table of Contents
```

Comment:
Isn't deleting this covered in another PR or ticket somewhere?

It looks like it is, but it's showing up here because this is chained on that PR? Make sure you set the proper base branch, labels, etc. if I'm correct and this should be a chain.

Reply:
Its a freebie, the commit issue is referenced in the title. The base should be develop. Is there something I messed up? I thought the blocking relation would be appropriate.

Reply:
Sorry, my mistake. I thought this was already in a separate PR. With the new Zenhub changes you could try linking the freebie as an experiment, but that's not approved process.